xray/test/test_backends.py

```diff
@@ -651,6 +658,24 @@ def roundtrip(self, data, **kwargs):
                 yield ds
 
 
+@requires_netCDF4
+class NetCDF4ClassicViaNetCDF4DataTest(CFEncodedDataTest, TestCase):
```

Comment:
Is that preferable to running the complete set of tests against the round-trip netcdf4-classic?

I added the Only32BitTypes class back in to NetCDF4ClassicViaNetCDF4DataTest and it fixed the failure... (although it is now technically misnamed :))